src/display/annotation_layer.js

```diff
@@ -305,127 +306,35 @@ var TextAnnotationElement = (function TextAnnotationElementClosure() {
      * @returns {HTMLSectionElement}
      */
     render: function TextAnnotationElement_render() {
-      var rect = this.data.rect, container = this.container;
-
-      // Sanity check because of OOo-generated PDFs.
```

Comment:
Would you please mind explaining why the checks below are no longer necessary in the new code!?